features/manage-position/Create.tsx

```diff
@@ -159,6 +164,17 @@ const Create = () => {
       }
     };
 
+    const setBackingCollateralToMin = () => {
+      // By default set amount of collateral to the minimum required by the GCR constraint. This
+      // default is intended to encourage users to maximize their capital efficiency.
+      const minBackingCollateral = gcr * Number(tokens);
+      setCollateral(minBackingCollateral.toString());
```

Comment:
should we consider rounding this to 4 decimal places? it looks a bit nasty in the UI IMHO for these numbers:
![image](https://user-images.githubusercontent.com/12886084/90799816-0b7fd700-e314-11ea-8fcb-6cd27030433b.png)

Otherwise, you can round in the button or input field.

Reply:
I rounded it to 4